torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -13,7 +13,7 @@ namespace jit{
 char const * toString(OpCode op);
 std::ostream& operator<<(std::ostream& out, Instruction inst);
 namespace mobile {
-InterpreterState::InterpreterState(std::shared_ptr<Code> code) : code_(code) {
+InterpreterState::InterpreterState(std::shared_ptr<Code> code) : code_(std::move(code)) {
```

Comment:
Should we make this constructor take rvalue? `std::shared_ptr<Code>&& code`? That way the caller understands that the value maybe moved? Although all of this is self-contained, making it explicit might be better?

Reply:
As discussed offline, rvalue ref cannot be used here because the ownership of code may be shared out of InterpreterState.